client/app/feedback-form/feedbackForm.controller.js

```diff
@@ -50,6 +51,16 @@ angular.module('spkr.feedback-form', ['ngRoute'])
       })
     }
 
+    $scope.autoExpand = function(e) {
+        var element = typeof e === 'object' ? e.target : document.getElementById(e);
```

Comment:
I would put everything after 'element =' inside parenthesis to increase clarity.